src/AppInstallerCLICore/Workflows/ShellExecuteInstallerHandler.cpp

```diff
@@ -205,7 +206,7 @@ namespace AppInstaller::CLI::Workflow
             context.Reporter.Warn() << "Installation abandoned" << std::endl;
             AICLI_TERMINATE_CONTEXT(E_ABORT);
         }
-        else if (installResult.value() != 0)
+        else if (installResult.value() != 0 && (std::find(successCodes.begin(), successCodes.end(), static_cast<int>(installResult.value())) == successCodes.end()))

```

Comment:
>installResult.value() != 0 [](start = 17, length = 26)

Should 0 be included by default if codes are provided?  What if 0 is a failure for some installer?  I can understand the confusion that this could create, so it might be better to have a separate flag in the manifest for making 0 a failure value.